test/models/foster_test.rb

```diff
@@ -1,7 +1,18 @@
 require "test_helper"
 
 class FosterTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'valid foster' do
+    foster = Foster.new(email: 'johndoe@mail.com', street: '123')
+    assert foster.valid?
+  end
+
+  test 'invalid without street' do
```

Comment:
I would update the description to say something like `test 'does not add a foster with an empty address' do`.  It will be more readable when the tests fail.